packages/client/hmi-client/src/services/charts.ts

```diff
@@ -146,6 +141,7 @@ export const createForecastChart = (
 	samplingLayer: ForecastChartLayer | null,
 	statisticsLayer: ForecastChartLayer | null,
 	groundTruthLayer: ForecastChartLayer | null,
+	annotationLayers: any[],
```

Comment:
This can be an array of any vegalite spec...ideally keep it to annotations only.  I needed 2 layers for a rule layer and a text layer

Reply:
I am hesitant to make things too open ended, because then you might as well extract the json and just do things after-the-fact, e.g:

```
const blah = createForecastChart(....)
blah.layers.push({
});
```

Reply:
That makes sense, maybe Ill keep the function as is and do what you said here